CHANGELOG

```diff
@@ -1,6 +1,9 @@
 qtile next_version:
-    !!! extention subpackage renamed to extension !!!
-    !!! extentions option renamed to extensions !!!
+    !!! `extention` subpackage renamed to `extension` !!!
+    !!! `widget_defaults` and `extentions` options merged into `extension_defaults` !!!
```

Comment:
I see why it would be nice to unify these, but it might be better to pick a name that is neither specific to widgets or extensions. Particularly because widget_defaults will be the most prevalent for users.